liberica/src/components/Navbar.tsx

```diff
@@ -1,30 +1,9 @@
 import { PropsWithChildren } from "react";
-import { Button } from "./InputElements";
-import { FaHome } from "react-icons/fa";
-import { useNavigate } from "react-router-dom";
 
 export function Navbar(props: PropsWithChildren) {
     return (
-        <div
-            className="absolute bottom-0 flex w-max gap-3 bg-white p-2"
-            style={{
-                position: "fixed",
-                justifyContent: "space-between",
-                alignItems: "center",
-                zIndex: 1000,
-            }}
-        >
+        <div className="fixed bottom-0 z-auto flex max-h-14 w-dvw items-center justify-between gap-3 rounded-t-2xl bg-base p-2">
             {props.children}
         </div>
     );
 }
-
-export function HomeButton() {
-    const navigate = useNavigate();
-
-    return (
-        <Button onClick={() => navigate("/")}>
-            <FaHome />
-        </Button>
-    );
-}
```

Comment:
Why remove the home button?

Reply:
I am currently experimenting with the Navbar and couldn't find space for it (It also didn't fit the theme). I will put it back sometime later.